tests-performance/src/template-with-cli-args.cc

```diff
@@ -0,0 +1,67 @@
+#include <mata/inter-aut.hh>
+#include <mata/nfa.hh>
+#include <mata/mintermization.hh>
+#include <iostream>
+#include <fstream>
+#include <chrono>
+#include <string>
+#include <cstring>
+
+using namespace Mata::Nfa;
+
+int main(int argc, char *argv[])
+{
+	if (argc != 2) {
+		std::cerr << "Input file missing\n";
+		return EXIT_FAILURE;
+	}
+
+	std::string filename = argv[1];
+
+	std::fstream fs(filename, std::ios::in);
+	if (!fs) {
+		std::cerr << "Could not open file \'" << filename << "'\n";
+		return EXIT_FAILURE;
+	}
+
+	Mata::Parser::Parsed parsed;
+	Nfa aut;
+    Mata::StringToSymbolMap stsm;
+	const char* nfa_str = "NFA";
+	try {
+		parsed = Mata::Parser::parse_mf(fs, true);
+		fs.close();
+
+		if (parsed.size() != 1) {
+			throw std::runtime_error(
+				"The number of sections in the input file is not 1\n");
+		}
+		if (strncmp(parsed[0].type.c_str(), nfa_str, strlen(nfa_str)) != 0) {
```

Comment:
Then I can use `std::string::compare` instead of `strncmp` and `.size()` instead of `strlen` here. No need to fall back to C in this case.

Reply:
Ook. Makes sense.